Source/WebCore/page/EventSource.cpp

```diff
@@ -382,9 +382,7 @@ void EventSource::parseEventStreamLine(unsigned position, std::optional<unsigned
         if (!valueLength)
             m_reconnectDelay = defaultReconnectDelay;
         else {
-            // FIXME: Do we really want to ignore trailing junk here?
-            // FIXME: When we can't parse the value, should we really leave m_reconnectDelay alone? Shouldn't we set it to defaultReconnectDelay?
```

Comment:
Is there a WPT test covering the answer to this question?

Reply:
No but our behavior matches other browsers. I'll add such test and upstream it.

Reply:
https://github.com/web-platform-tests/wpt/pull/35249